local.c

```diff
@@ -1181,6 +1181,14 @@ static int read_device_name(struct iio_device *dev)
 		return 0;
 }
 
+static int cmp_iio_device_attr(const void *p1, const void *p2)
+{
+	const char *tmp1 = *(const char **)p1;
+	const char *tmp2 = *(const char **)p2;
+
+	return strcmp(tmp1, tmp2);
```

Comment:
is there a MAX value/definition for these so that strncmp() can be used ?

Reply:
I don't think so - this is the max path that the kernel knows about. @larsclausen or @pcercuei would know for sure.

Reply:
These strings will always be NULL-terminated, so strcmp() is fine.